### ISSUE TITLE
IINA Can not output from S/PDIF with original AC3 DTS DTS-HD sound track

### ISSUE THREAD

**Issue by jiangtiger** [lhc70000/iina#2023]
**System and IINA version:**

- macOS 10.13.4
- IINA 0.0.15.1

**Expected behavior:**

**Actual behavior:**

- Crash report:
```

```

- MPV log:
```

```

**Steps to reproduce:**

- [x] MPV does not have this problem.
Movisit and Blu-ray Player Pro can direct output the original DTS AC-3 sound track.

**How often does this happen?**

**Comment by inflation** [comment]
Please make sure you have enabled S/PDIF output in the settings:
![image](https://user-images.githubusercontent.com/2375962/46578506-17887700-c9d0-11e8-9932-77be13be9f01.png)

**Comment by jiangtiger** [comment]
Yes I have enable this settings, but the 5.1 channels sound track was mixed as  2 channels decode from my speaker

**Comment by inflation** [comment]
@jiangtiger Then see #1472 and enable audio channels in the Advanced settings. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pleasejust** [comment]
Looking at this image now, the new settings page actually looks nice.
